easybuild/easyconfigs/d/DIRAC/DIRAC-24.0-intel-2023a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'DIRAC'
+version = '24.0'
+
+homepage = 'http://www.diracprogram.org'
+description = "DIRAC: Program for Atomic and Molecular Direct Iterative Relativistic All-electron Calculations"
+
+toolchain = {'name': 'intel', 'version': '2023a'}
+toolchainopts = {'usempi': True, 'oneapi': False}
+
+source_urls = ['https://zenodo.org/records/10680561/files/']
+sources = ['DIRAC-%(version)s-Source.tar.gz']
+checksums = ['8943e4a24dbacde3ce3daacc2a6ecebc282702a78fa12638e919b78667b293f5']
+
+builddependencies = [('CMake', '3.26.3')]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('HDF5', '1.14.0')
+]
+
+configopts = '-DMKL_FLAG=parallel '
```

Comment:
This has `parallel`, #19960 has `off` for same configuration option.

Which one is the correct approach?

Reply:
Thank you for your feedback. The parallel MKL flag is only used in the two Intel compilations. It is set to "off" in the foss compilation. This is the desired approach.